src/www/app/outline_server_repository/server.ts

```diff
@@ -78,6 +84,16 @@ export class OutlineServer implements Server {
 
   async connect() {
     try {
+      if (this.isDynamic) {
+        this.sessionConfig = await (
```

Comment:
Actually, thinking about this more - we sure we want to do this? It might be nice to have the session hang in memory in case the original online config enpoint gets blocked

Reply:
Okay so maybe later then. Seems reasonable.